common/config/azure-pipelines/jobs/fast-ci.yaml

```diff
@@ -27,12 +27,49 @@ pr:
       - "**/CHANGELOG.md"
 
 jobs:
-  - template: ci-core.yaml
+- job:
+  # variables:
+  # - name: linuxPool
+  #   value: 'iModeljs Linux'
+  # - name: mac_pool
+  #   value: iModelTechCI
+  # - name: win_pool
+  #   value: iModelTechCI
+
+  strategy:
+    matrix:
+      "Windows_Node_18":
+        OS: windows-latest
+        platform: Windows_NT
+        name: $(win_pool)
+      "Linux_Node_18":
+        OS: ubuntu-latest
+        platform: Linux
+        name: $(linux_pool)
+      "MacOS_Node_18":
+        OS: macOS-latest
+        platform: Darwin
+        name: $(mac_pool)
+
+  pool:
+    name: $(name)
+    demands:
+      - Agent.OS -equals $(platform)
+
+  timeoutInMinutes: 60
+
+  workspace:
+    clean: all
+
+  steps:
+  - checkout: self
+    clean: true
+  - template: ../templates/core-build.yaml
     parameters:
```

Comment:
i think you need another param for buildIos like the old ci-core had

![image](https://user-images.githubusercontent.com/11051042/202255227-4b46a6df-2126-41c6-9b86-fdd83ecb34f3.png)

Reply:
i dont think it should be always true, only when the agent is a mac. no xcode on windows or linux

Reply:
Ah ok. It was set to true in original fast-ci.yaml, but I understand now